app/dao/organisations_dao.py

```diff
@@ -0,0 +1,23 @@
+from app import db
```

Comment:
trying to keep it in line with other daos, if it's an issue I can change it back